MLOps/intro-to-ml-model-deployment-and-management/intro-to-ml-model-deployment-and-management.md

```diff
@@ -0,0 +1,1010 @@
+
+## Task 6: Monitor MLOps Deployed Model
+
+1\. In the **Deployments** table on the hydraulic system page, on the **hydraulic_system_2** row, click on **3 dots** actions and then click on **Monitoring**.
+
+The grafana dashboard with all the monitoring visualizations of our model deployment will be shown. Those visualizations will be updated every 30s by default, but you can adjust this number if you want the visualizations updated faster. For instance, this number has been updated to 10s.
+
+You can see metadata for alerts and column types:
+
+![mlops-deployed-model-monitoring-1](./assets/mlops-deployed-model-monitoring-1.jpg)
+
+2\. Scroll down the monitoring dashboard to see all the visualizations. 
+
+You can see a visualization for the deployed model's scoring latency:
+
+![mlops-deployed-model-monitoring-2](./assets/mlops-deployed-model-monitoring-2.jpg)
+
+You can see a scatter plot, histogram and heatmap visualization for psa_bar:
+
+![mlops-deployed-model-monitoring-3](./assets/mlops-deployed-model-monitoring-3.jpg)
+
+![mlops-deployed-model-monitoring-4](./assets/mlops-deployed-model-monitoring-4.jpg)
+
+As you scroll down the visualizations, you will see the samples collected increasing and visualizations for each column in the test set.
+
+3\. Keep scrolling until you get to the target column, which is **cool_cond_y** in our case:
+
+4\. We have a visualizations for cool_cond_y.3, cool_cond_y.20 and cool_cond_y.100. Let's look at cool_cond_y.3:
+
+![mlops-deployed-model-monitoring-5](./assets/mlops-deployed-model-monitoring-5.jpg)
+
+![mlops-deployed-model-monitoring-6](./assets/mlops-deployed-model-monitoring-6.jpg)
+
+What is happening with the variables used when making predictions? How are the visualization graphs for the target column?
+
+## Task 7: Challenge
+
+### Deploy New Driverless AI Model using MLOps
+
+You just learned how to deploy a Driverless AI model that predicts hydraulic cooling condition. You may have some other problems you are working on solving. Try deploying a new Driverless AI model from a different Diverless AI project using MLOps.
+
+### Perform Predictions via Programmatic Score Requests
+
+So far we have covered making score requests in the command line with curl and through a live code data recipe in Python, but maybe you have an application you want to perform the predictions in? If there is an application that you have built, you could build a Driverless AI model using the dataset, then make score requests using the programming language your application was written in: Java, Python, C++, etc.
+
+## Next Steps
+
+- Check out these webinars that dive into how to productionize Driverless AI models using MLOps:
+    - H2O Webinar: [Getting the Most Out of Your Machine Learning with Model Ops](https://www.brighttalk.com/service/player/en-US/theme/default/channel/16463/webcast/418711/play?showChannelList=true)
+
+## Appendix A: Share Driverless AI Credit Card Project with MLOps
```

Comment:
QQ: How does this section play into the larger tutorial?  That credit card use-case seems almost orthogonal to the current tutorial.  The steps are almost identical (except for having to upload the dataset), except it's very concise and less verbose.

I wonder if moving this to some sort of "Quick Start" tutorial instead of making a long tutorial even longer has any benefit.  Just wondering.

Reply:
@orendain I was debating whether to keep this appendix section or not. I was leaning toward removing it. I can commented it out for now in case the team later wants to leverage it for an MLOps quickstart tutorial.